<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumberProviderDialog</class>
 <widget class="QDialog" name="NumberProviderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Number provider</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="MultiPageWidget" name="multipagewidget">
     <widget class="QWidget" name="exact">
      <property name="windowTitle">
       <string>Exact number</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Value:</string>
         </property>
         <property name="buddy">
          <cstring>exactSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="OptionalSpinBox" name="exactSpin"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="uniform">
      <property name="windowTitle">
       <string>Uniform distribution</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Minimun:</string>
         </property>
         <property name="buddy">
          <cstring>minValueInput</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="NumberProvider" name="minValueInput"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Maximum:</string>
         </property>
         <property name="buddy">
          <cstring>maxValueInput</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="NumberProvider" name="maxValueInput"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="binomial">
      <property name="windowTitle">
       <string>Binomial distribution</string>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Number of trials (n):</string>
         </property>
         <property name="buddy">
          <cstring>numberInput</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="NumberProvider" name="numberInput"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Success probability (p):</string>
         </property>
         <property name="buddy">
          <cstring>probInput</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="NumberProvider" name="probInput"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="score">
      <property name="windowTitle">
       <string>Scoreboard value</string>
      </property>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Target</string>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QRadioButton" name="contextRadio">
            <property name="text">
             <string>Context:</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="contextCombo"/>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="fixedRadio">
            <property name="text">
             <string>Fixed:</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="nameEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Score:</string>
         </property>
         <property name="buddy">
          <cstring>scoreEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="scoreEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Scale:</string>
         </property>
         <property name="buddy">
          <cstring>scaleSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="OptionalSpinBox" name="scaleSpin"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="storage">
      <property name="windowTitle">
       <string>Storage</string>
      </property>
      <layout class="QFormLayout" name="formLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Storage ID:</string>
         </property>
         <property name="buddy">
          <cstring>storageEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="storageEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>NBT path:</string>
         </property>
         <property name="buddy">
          <cstring>nbtPathEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="nbtPathEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="enchantLevel">
      <property name="windowTitle">
       <string>Enchantment level</string>
      </property>
      <layout class="QFormLayout" name="formLayout_7">
       <item row="0" column="0" colspan="2">
        <widget class="DialogDataButton" name="enchantLevelBtn">
         <property name="text">
          <string>Amount...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NumberProvider</class>
   <extends>QFrame</extends>
   <header>numberprovider.h</header>
  </customwidget>
  <customwidget>
   <class>DialogDataButton</class>
   <extends>QWidget</extends>
   <header>dialogdatabutton.h</header>
  </customwidget>
  <customwidget>
   <class>MultiPageWidget</class>
   <extends>QWidget</extends>
   <header>multipagewidget.h</header>
   <addpagemethod>addPage</addpagemethod>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OptionalSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>optionalspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NumberProviderDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NumberProviderDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contextRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>contextCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fixedRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>nameEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
